listener/src/event_handler/flux_transition_handler.rs

```diff
@@ -179,6 +185,79 @@ pub mod test {
         assert!(result.is_err());
     }
 
+    #[test]
```

Comment:
Could add a test for when no state exists?

Reply:
Sure thing! I'm on it!

Reply:
Actually, I am not sure this is going to work here since a pre-existing state is needed in order to "process_event"

https://github.com/gnosis/dex-services/blob/7849c453507b7fe359b782fab76e4a4417f687ce/listener/src/event_handler/flux_transition_handler.rs#L64-L67

Unless you are hinting at the fact that this `ok_or_else` should be tested.. If so, I would argue this is not part of this issue.

Reply:
I think it would be useful to document/test what happens if we receive such an event but find no state for the specified hash (or there is no state at all in the DB)

The test should make sure that we return an error in that case. 

The fact that it’s is just testing an ok_or is an implementation detail that might change in the future.

Reply:
Yes I like this will make a separate PR with this and any other tests like this today!